CMakeLists.txt

```diff
@@ -7,7 +7,11 @@ include(ExternalProject)
 set(CMAKE_C_COMPILER arm-linux-gnueabihf-gcc)
 set(CMAKE_EXE_LINKER_FLAGS -static)
 
-enable_testing()
```

Comment:
I wonder if `BUILD_TESTING` could be used instead of `WITH_TEST` (https://cmake.org/cmake/help/v3.15/command/enable_testing.html)?

Reply:
It seems to work! See 5f43cc1225b3dd34cc47aa1e0013b8b6badaa938.